### PR TITLE
fix format

### PR DESCRIPTION
**Pull request by selmaillig**

### CODE REVIEW

File: prod-requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+grpcio==1.18.0
+grpcio-tools==1.18.0
```
Comment (alexmorten) [inline prod-requirements.txt]: I'm not sure if we need the grpc-tools in the prod-requirements (I think we only need them to generate the python files from the proto repo?)

Comment (Animuslupus) [reply prod-requirements.txt]: No the grpcio is the pip-package for the grpc-python library (see [grpcio-pip](https://pypi.org/project/grpcio/)), which is used in main.py to start the server.

Comment (alexmorten) [reply prod-requirements.txt]: I agree that we definitely need the `grpcio`package in runtime, but my gut feeling is that the `grpc-tools` (just judging by the name) could be necessary for generation purposes only and not for starting the server.

That's however something we just have to try

Comment (Animuslupus) [reply prod-requirements.txt]: You are right. `grpcio-tools` is just used to generate the python classes representing the protobuffer. :+1:

Comment (alexmorten) [reply prod-requirements.txt]: It doesn't shrink the image size dramatically, but it's a start 😅

**Comment by alexmorten** [comment]
Do you want to add travis in this branch too? :)

**Pull request merged**